source/guide/3.2/webui-blueprints-page.md

```diff
@@ -13,15 +13,31 @@ terminology_link: reference-terminology.html
 When clicking on the `Blueprints` tab and choosing a blueprint you will be able to choose one of the following:
 
 # Topology
-See the definition [here]({{page.terminology_link}}#topology).<br/>
+The [Topology]({{page.terminology_link}}#topology) is an application’s graph of nodes and their relationships, which describes the lifecycle events or other operations that each node and relationship exposes for use in workflows.<br>
+Each of the blueprint nodes is displayed as a square container, which can contain another nodes. Each of the nodes has a title describing its name, and an icon to indicate the node type.<br>
```

Comment:
each of the blueprint -> blueprint's
contain another -> contain other
each of the nodes -> each node
the node type -> the node's type